frontend/api/answers/[id].ts

```diff
@@ -13,6 +13,29 @@ export default async function (req: VercelRequest, res: VercelResponse) {
     } else {
       return res.json(result);
     }
+  } else if (req.method === 'PATCH') {
+    const value = req.body.value;
+    const updateToken = req.body.updateToken;
+    const existingResult = await prisma.result.findUnique({
+      where: { id: resultId },
+    });
+
+    if (existingResult === null) {
+      return res.status(404).json({ error: `Result ${resultId} not found` });
+    }
+    if (existingResult.updateToken !== updateToken) {
+      return res.status(403).json({ error: `Invalid update token` });
+    }
+
+    const result = await prisma.result.update({
+      where: { id: resultId },
+      data: {
+        value: value,
+        source: req.body.embedSourceUrl,
+        embedSourceUrl: req.body.embedSourceUrl,
```

Comment:
Mělo by to umožnit updatovat jenom tu `value`. A pak by to chtělo ještě přidat to `updatedAt`,

Reply:
upraveno a `updatedAt` je tam taky